phabricatoremails/mail.py

```diff
@@ -163,16 +175,24 @@ def send(self, emails: List[OutgoingEmail]):
                 f'[{email.to}] Sending "{email.template_path}" for "{email.subject}"'
             )
 
-            destination = self._send_to if self._send_to else email.to
-            mime_message = email.to_mime_message(
-                self._from_address, include_target_in_subject=self._send_to is not None
-            )
-
-            # send_raw_email() is used instead of send_email() because it provides
-            # greater flexibility, such as specifying the `Date` header (which isn't
-            # possible with `send_email()`).
-            self._client.send_raw_email(
-                RawMessage={"Data": mime_message.as_string()},
-                Source=self._from_address,
-                Destinations=[destination],
-            )
+            try:
+                destination = self._send_to if self._send_to else email.to
+                mime_message = email.to_mime_message(
+                    self._from_address,
+                    include_target_in_subject=self._send_to is not None,
+                )
+
+                # send_raw_email() is used instead of send_email() because it provides
+                # greater flexibility, such as specifying the `Date` header (which isn't
+                # possible with `send_email()`).
+                self._client.send_raw_email(
+                    RawMessage={"Data": mime_message.as_string()},
+                    Source=self._from_address,
+                    Destinations=[destination],
+                )
+            except Exception as e:
```

Comment:
Can this be more specific?

Reply:
Good question, it depends on the outcome of [this conversation](https://github.com/mozilla-conduit/phabricator-emails/pull/20/files#r508783345)